applications/omnipresence/src/omnip_util.erl

```diff
@@ -35,3 +37,19 @@ is_valid_uri(Uri) ->
         [_User, _Host] -> 'true';
         _ -> 'false'
     end.
+
+-spec request_probe(binary(), binary()) -> 'ok'.
+request_probe(Package, User) ->
+    case binary:split(User, <<"@">>, ['global']) of
+        [Username, Realm | _] -> request_probe(Package, Username, Realm);
+        _Other -> lager:warning("request probe for ~p failed", [_Other])
+    end.
+
+-spec request_probe(binary(), binary(), binary()) -> 'ok'.
+request_probe(Package, Username, Realm) ->
+    API = [{<<"Event-Package">>, Package}
+          ,{<<"Username">>, Username}
+          ,{<<"Realm">>, Realm}
+	   | kz_api:default_headers(?APP_NAME, ?APP_VERSION)
```

Comment:
Great, that slow/buggy elvis script isn't even catching the TABs we ask it to catch..